File: Oxiyopal.Models/Bodega.cs
using Oxiyopal.Models.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace Oxiyopal.Models
{
    public class Bodega: EntityBase<int>
    {
        public string NombreBodega { get; set; }
        public string Direccion { get; set; }

        public int CiudadId { get; set; }
        public Ciudad Ciudad { get; set; }

       
        public IList<Ubicacion> Ubicaciones { get; set; }
    }
}

File: Oxiyopal.WebUI/ClientApp/src/app/models/Bodega.ts
import { EntityBase } from './Base/EntityBase';
import { Ubicacion } from './Ubicacion';
import { Ciudad } from './Ciudad';

export interface Bodega extends EntityBase<number> {
    nombreBodega: string;
    direccion: string;

    ciudadId: number;
    ciudad: Ciudad;


    ubicaciones: Ubicacion[];
}

